src/main/java/com/google/sps/data/DBReferences.java

```diff
@@ -0,0 +1,10 @@
+package com.google.sps.data;
```

Comment:
You should put all capstone code in our own package names.

Reply:
What would that look like? Something like `com.google.meltingpot.data`?

Reply:
let's all use com.google.sps.meltingpot.data. I think they are using "sps" to mean STEP interns here.

Reply:
Yup, done in 24c6d9efcaab9c9ec978bcd9d72e9c37aa5ea03d